spec/restaurant_spec.rb

```diff
@@ -0,0 +1,12 @@
+require './docs/restaurant'
+
+describe Restaurant do
+  it 'lists the dishes with their prices' do
+    # Arrange
+    restaurant = Restaurant.new
+    # Act
+    show_menu = restaurant.menu
+    # Assert
+    expect(show_menu).to eq("Here is a list of our dishes: #{subject.list_dishes}")
```

Comment:
You don't always need to use Arrange Act Assert. 'Act' is something that changes the world; e.g. docking a bike into a station changes where the bike is. But printing isn't like that. You can therefore refactor to `expect(restaurant.menu)` etc. For extra brownie points, there's a way to test what is being printed to 'stdout' (standard output): https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/output-matcher (Hint - you'll need to take newlines into consideration with printing.)